tests/db_tests/test_queries.py

```diff
@@ -334,8 +334,8 @@ def test_transaction_retry_wrapper_wrapper_max_retries(mocker, test_client, capl
             decorated_function()
 
     assert mocked_function.call_count == max_retries
-    assert caplog.messages[-1] == "Number of max retries exceeded for function 'mocked_function'"
-    assert patched_time_sleep.call_count == 2
+    assert caplog.messages[-1] == ("Number of max retries exceeded for function 'mocked_function'")
+    assert patched_time_sleep.call_count == 4
```

Comment:
Nit picky but it may be clearer to assert that `mocked_function` is called 5 times, rather than asserting on the number of sleeps? Or perhaps both may be best